### PR TITLE
[1LP][RFR] C&U broken for cloud

### PR DESCRIPTION
**Pull request by nachandr**
Adding 2 BZ blockers through this BZ.

1)BZ 1531600, which is relevant to all supported infra and cloud providers for 59 only.
2)BZ 1531554 ,which is relevant to all supported infra and cloud providers for 58 only.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit fbfbe2cac4fcbc88aa540393af2bfeb059b0ec74:

`cfme/tests/intelligence/reports/test_validate_chargeback_report.py`:
- :bangbang: Line 32:32: **E131** *continuation line unaligned for hanging indent*
- :bangbang: Line 34:53: **E999** *SyntaxError: invalid syntax*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/intelligence/reports/test_validate_chargeback_report.py [hunk 1]
```diff
@@ -29,6 +29,9 @@
                                BZ(1468729, forced_streams=["5.9"]),
                                BZ(1511099, forced_streams=["5.7", "5.8"],
                                   unblock=lambda provider: not provider.one_of(GCEProvider))]),
+                               BZ(1531554, forced_streams=["5.8"],
+                                  unblock=lambda provider: not provider.one_of(GCEProvider,
+                                    AzureProvider))]),
```
Comment (mshriver) [inline cfme/tests/intelligence/reports/test_validate_chargeback_report.py]: Comma is outside of the `meta` statement

**Comment by dajoRH** [comment]
Lint report for commit 874fcb03a03d8fe8bc69f5dcb9a5961095faf674:

`cfme/tests/intelligence/reports/test_validate_chargeback_report.py`:
- :bangbang: Line 32:32: **E131** *continuation line unaligned for hanging indent*
- :bangbang: Line 33:96: **E999** *SyntaxError: invalid syntax*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/intelligence/reports/test_validate_chargeback_report.py [hunk 2]
```diff
@@ -27,8 +28,11 @@
     pytest.mark.tier(2),
     pytest.mark.meta(blockers=[BZ(1433984, forced_streams=["5.7", "5.8", "upstream"]),
                                BZ(1468729, forced_streams=["5.9"]),
+                               BZ(1531600, forced_streams=["5.9"]),
                                BZ(1511099, forced_streams=["5.7", "5.8"],
-                                  unblock=lambda provider: not provider.one_of(GCEProvider))]),
+                                  unblock=lambda provider: not provider.one_of(GCEProvider)),
+                               BZ(1531554, forced_streams=["5.8"],
```
Comment (mshriver) [inline cfme/tests/intelligence/reports/test_validate_chargeback_report.py]: Looks like this bug is relevant for infra providers as well, as the UI element is missing in PRT screenshots for rhv41, vsphere*, etc in 5.8 results.

**Comment by mshriver** [comment]
Proposing rc-regression-fix label for stabilization

**Pull request merged**